src/main/clm_initializeMod.F90

```diff
@@ -337,7 +337,7 @@ subroutine initialize2(ni,nj)
     end if
 
     ! Pass model timestep info to FATES
-    call CLMFatesTimesteps()
+    if (use_fates) call CLMFatesTimesteps()
```

Comment:
Without the `if`, the code populated fates params with NaNs.

This change gets me past this issue, but a new one has surfaced.